.gitignore

```diff
@@ -35,4 +35,8 @@ override.tf.json
 
 # Ignore CLI configuration files
 .terraformrc
-terraform.rc
\ No newline at end of file
+terraform.rc
+
+# Ignore IDE directories
+.idea
+.idea/*
```

Comment:
@hunterkepley I think this can just be `.idea/` to ignore the directory as-per https://git-scm.com/docs/gitignore